### PR TITLE
Greta alisauskiene/feature/t 2 verify all user roles

### PR DESCRIPTION
**Pull request by GretaAl**

### CODE REVIEW

File: cypress/integration/T2.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+describe('Sourcebooks login', function() {
+
```
Comment (DavidV44) [inline cypress/integration/T2.js]: Unneeded empty line

File: cypress/integration/login.js [hunk 2]
```diff
@@ -14,14 +14,19 @@ describe('Sourcebooks login', function() {
...
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav').find('li').should('have.length', 2);
+
+
```
Comment (DavidV44) [inline cypress/integration/login.js]: Also unneeded empty line, there usually are no cases that 2 empty lines would be needed

File: cypress/integration/login.js [hunk 3]
```diff
@@ -14,14 +14,19 @@ describe('Sourcebooks login', function() {
...
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav').find('li').should('have.length', 2);
+
+      
+        cy.get('.calendar--today').find('span').contains('1');
+    
+
```
Comment (DavidV44) [inline cypress/integration/login.js]: Same as above

**Review by DavidV44 — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js [hunk 4]
```diff
@@ -14,14 +14,15 @@ describe('Sourcebooks login', function() {
         cy.get('[id="loginForm.userId"]').click({force:true});
         cy.get('[aria-label="Demo User"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: Please finish T1 before merging T2

File: cypress/integration/login.js [hunk 5]
```diff
@@ -14,14 +14,15 @@ describe('Sourcebooks login', function() {
...
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav').find('li').should('have.length', 2);
+        cy.get('.calendar--today').find('span').contains(new Date().getDate());
```
Comment (nikolaj-tolkaciov) [inline cypress/integration/login.js]: Please finish T1 before merging T2

**Review by nikolaj-tolkaciov — CHANGES_REQUESTED** [review]

File: cypress/integration/login.js (hunk 4 above)
Comment (GretaAl) [reply cypress/integration/login.js]: Done

File: cypress/integration/login.js (hunk 5 above)
Comment (GretaAl) [reply cypress/integration/login.js]: Done

**Pull request merged**